wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/implementation/QuantityValueImpl.java

```diff
@@ -49,6 +49,31 @@
 	 */
 	private final JacksonInnerQuantity value;
 	
+	/**
+     * Constructor.
+     *
+     * @param numericValue
+     *            the numeric value of this quantity
+     * @param lowerBound
+     *            the lower bound of the numeric value of this quantity or null
+     *            if not set
+     * @param upperBound
+     *            the upper bound of the numeric value of this quantity or null
+     *            if not set
+     * @param unit
+     *            the unit of this quantity, or null if there is no
+     *            unit
+     */
+    public QuantityValueImpl(
+            BigDecimal numericValue,
+            BigDecimal lowerBound,
+            BigDecimal upperBound,
+            ItemIdValue unit) {
+        super(JSON_VALUE_TYPE_QUANTITY);
+        this.value = new JacksonInnerQuantity(numericValue, lowerBound,
+                upperBound, unit);
+    }
```

Comment:
There seems to be a tab vs space problem.

Reply:
Thank you for the fix! However, it seems you included a lot of unrelated files in this change.